### ISSUE TITLE
SkypeWeb don't receive authorization requests

### ISSUE THREAD

**Issue by rand256** [miranda-ng/miranda-ng#1781]
When a contact tries to send me an authorization request to be able to send me messages, it doesn't make it into miranda at all. Only after I authorize it in web.skype.com it appears in miranda like this:

>  19.01.2019 12:19:36: Unknown event, please send this text for developer: "Hi, <my user name>, I'd like to add you as a contact."

Both contacts are old, using old skype usernames and not the modern microsoft ones (if that matters).

**Comment by georgehazan** [comment]
a network log will be appreciated, SkypeWeb + core.
https://wiki.miranda-ng.org/Network_log

**Comment by rand256** [comment]
There's too much private data including full contact list and tokens in the log. But looking at it I can see the differences depending on different initial situation:

- S1: contact sending auth request is already present on local contact list of miranda, but not authorized at skype.
result => nothing in the log containing name of that contact.

- S2: contact sending auth request is missing on local contact list of miranda and not authorized at skype, but auth request was already sent when miranda was offline.
result => there's a line in the log like this:
> {"invite_list":[{"mri":"8:remote_username","invites":[{"message":"Hi, my_username, I'd like to add you as a contact.","time":"2019-01-21T15:36:59Z"}],"displayname":"remote_username","avatar_url":"https://avatar.skype.com/v1/avatars/remote_username?auth_key=-893551981"}]}

In miranda a new contact was auto-created with that name, but no signs of auth request is visible. Incoming messages from the contact are also missing.

- S3: contact sending auth request is missing on local contact list of miranda and not authorized at skype, and there are no pending offline auth requests.
result => no signs of "remote_username" in logs (in plain text at least), nothing in miranda GUI.

To summarize:
It seems there's completely no way to know if a new contact tries to add you while you are in miranda.

**Comment by georgehazan** [comment]
fixed a while ago, now invite_list is processed
